sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/Generated/ConfidentialLedgerClient.cs

```diff
@@ -870,6 +1200,36 @@ public virtual Response GetUser(string userId, RequestContext context = null)
         /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="userId"/> or <paramref name="content"/> is null. </exception>
         /// <exception cref="ArgumentException"> <paramref name="userId"/> is an empty string, and was expected to be non-empty. </exception>
+        /// <example>
+        /// This sample shows how to call CreateOrUpdateUserAsync with required parameters and properties.
+        /// <code>
+        /// var credential = new DefaultAzureCredential();
+        /// var endpoint = new Uri("https://my-account-name.azure.com");
+        /// var client = new ConfidentialLedgerClient(endpoint, credential);
+        /// var data = new {
+        ///     assignedRole = "Administrator",
```

Comment:
How did we choose to populate the example value with "Administrator" instead of "AssignedRole"?

Reply:
This is an enum type, see the [swagger definition](https://github.com/Azure/azure-rest-api-specs/blob/f4d9fd5c6a0b8bb34a881728d2e8db28069f297a/specification/confidentialledger/data-plane/Microsoft.ConfidentialLedger/stable/2022-05-13/confidentialledger.json#L663-L675). Here we choose the first value, e.g., `Administrator`.